### ISSUE TITLE
[4.0.0-rc-2] Custom Fields with default positon (blank) do no appear in "Editor > Options > Custom"

### ISSUE THREAD

**Issue by BlakesHeaven** [bludit/bludit#1432]
### Describe your problem
In version 4 (e.g. RC2) for Custom Fields, is the (optional) 'position' variable still supported?
When the value is empty it should cause fields to appear in "Editor > Options > Custom"; however, they do not - they all appear in the "top" position.
There appears to be no 'Custom' option under "More Options"

Whether "Position" is supported or not, a secondary issue exists... the text editor height is too small when several custom fields are used and all are displayed in the "Top" position.
![image](https://user-images.githubusercontent.com/37149377/163734583-5696ceae-dd57-4887-a9b5-7425cf613566.png)

### Steps to reproduce the problem
Add these custom fields and create new content - all except the last one should appear in "Editor > Options > Custom"; however, they do not - they all appear in the "top" position.
```{
    "EventDate": {
        "label": "Event Date Time",
        "type": "string",
        "placeholder": "yyyy-mm-dd HH:MM",
        "tip": "Enter both the event date and time."
    },
    "VenueLocation": {
        "label": "Venue Location",
        "type": "string",
        "placeholder": "Venue Name & City\/Town"
    },
    "EventDuration": {
        "label": "Event Duration",
        "type": "string",
        "placeholder": "e.g. X hours Y mins"
    },
    "OpenDate": {
        "label": "Opening Date",
        "type": "string",
        "placeholder": "yyyy-mm-dd",
        "tip": "Tickets become available for ordering on this date."
    },
    "CloseDate": {
        "label": "Closing Date",
        "type": "string",
        "placeholder": "yyyy-mm-dd",
        "tip": "The last day tickets can be ordered."
    },
    "Cost": {
        "label": "Event Cost",
        "type": "string",
        "placeholder": "0.00",
        "tip": "Do not include the currency sign as it will be added automatically."
    },
    "Show": {
        "label": "Include in summary on parent",
        "type": "bool",
        "default": "true",
        "tip": "Deselect to remove the summary of this child page from its Parent.",
        "position": "top"
    }
}
```

### Bludit version
Version v4 RC2

**Comment by dignajar** [comment]
I added the following:
- Check custom fields JSON format before saving the changes
- Fix issues with booleans
- Positions supported: `top`, `bottom`, `menu`

This is an example for `"position": "menu"` 
![Screenshot 2022-05-03 at 19 43 31](https://user-images.githubusercontent.com/507352/166509208-2c9484d8-d2de-4c69-8e2e-d9f36b0c5b2a.png)

**Comment by dignajar** [comment]
Fixed, please check the new release v4.0.0 rc3.
Feel free to open the ticket if the issue is still there, if a new issue please open a new ticket.

**Comment by BlakesHeaven** [comment]
Thanks @dignajar
